### DISCUSSION TITLE
Not all conversations are shown on conversations tab

### DISCUSSION THREAD

**Discussion by gentlee** [community/community#62629] [Pull Requests]
### Select Topic Area

Bug

### Body

Please check this one https://github.com/101-Edu/Aktiv-React-Native/pull/6114/files

Conversations tab doesn't show all conversations in this PR, so we need to go to "File changed" tab to answer and resolve all conversations.

**Comment by ZeeshanMukhtar1** [comment]
Hello @gentlee ,

Thank you for bringing this issue to our attention. I've reviewed the provided link (https://github.com/101-Edu/Aktiv-React-Native/pull/6114/files) to understand the problem you are facing with the Conversations tab in the pull request.

If the Conversations tab is not showing all conversations in the PR, and you need to go to the "File changed" tab to answer and resolve them, this might be caused by various factors. Let's go through some troubleshooting steps to address this issue:

1 👍🏻  **Refresh the Page**: Sometimes, issues with GitHub's interface can be resolved by simply refreshing the page. Try refreshing your browser page and see if the Conversations tab starts displaying all the conversations.

2 👍🏻 **Check for Filters**: Ensure that there are no filters applied to the Conversations tab that might be hiding some of the conversations. Look for any filter options and reset them to default if necessary.

3 👍🏻 **Review GitHub Settings**: Verify the repository settings to ensure that there are no custom configurations affecting the display of conversations in pull requests. Check both repository settings and any GitHub Action workflows that might influence the Conversations tab.

4 👍🏻 **Check Permission Settings**: Ensure that you have the appropriate permissions to view all conversations in the pull request. If your access level is restricted, it might prevent you from seeing certain conversations.

5 👍🏻 **Browser Extensions:** Try accessing the pull request in an incognito or private browsing window to check if any browser extensions or plugins are causing conflicts with the Conversations tab.

6 👍🏻 **Clear Cache and Cookies**: Clear your browser's cache and cookies, as outdated data can sometimes cause issues with website functionality.

7 👍🏻 **Report to GitHub Support:** If the problem persists after trying the above steps, consider reaching out to GitHub support for further assistance. You can submit a ticket or post in the GitHub Community for help from the GitHub team and the broader community.

Please try these steps and let me know if the issue gets resolved. If you have any additional information or error messages related to this problem, please share them so we can investigate further. I'm here to help you resolve this issue and ensure smooth collaboration on your pull request.
